hash/pce.xml

```diff
@@ -4502,4 +4502,29 @@ https://mametesters.org/view.php?id=6597
 		</part>
 	</software>
 
+	<software name="acardduo" supported="no">
+		<description>Arcade Card Duo</description>
+		<year>1994</year>
+		<publisher>NEC</publisher>
+		<part name="cart" interface="pce_cart">
+			<feature name="slot" value="acard_duo" />
+			<dataarea name="rom" size="262144">
+				<!-- bios not included -->
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Lacking a BIOS means this option cannot be emulated. You shouldn't guess, given your past history makes me wonder if you even tested all this stuff.

Reply:
The Arcade Card Duo only contains the RAM needed for the Arcade titles as the Duo/Super CD System² came with the System Card 3.0 inbuilt

Reply:
That again implies the Arcade Card Duo is an option slot for a base system there's no real handling for. 
I also have my doubts wrt this kind of fake entry in the SW list, looks an unnecessary bridge for something doable with `-listslots` alone.

Reply:
I see no reason why not to have it, as it'd take up the cart slot on the console much like RAM carts for like the Saturn

Reply:
Well it doesn't do anything to begin with, it needs a base machine for the Duo that internalizes the CD System Card and make sure it overrides the mapping if a card is present, or overlays in the case you want Arcade stuff.
https://nicole.express/2020/why-do-you-need-an-arcade-card.html